### PR TITLE
utilizing better reward -> much smoother

### PR DESCRIPTION
**Pull request by nathanjzhao**

### CODE REVIEW

File: convert.py [hunk 1]
```diff
@@ -1,95 +1,71 @@
...
 import jax
 import flax
 import tensorflow as tf
 import tensorflowjs as tfjs
 from train import ActorCritic
-import tensorflow_probability as tfp
 import numpy as np
 
+# IMPORTANT !
+# SET TF_USE_LEGACY_KERAS=1
```
Comment (budzianowski) [inline convert.py]: add why

File: environment.py [hunk 2]
```diff
@@ -203,7 +203,9 @@ def compute_reward(
         is_healthy = jnp.where(state.q[2] < min_z, 0.0, 1.0)
         is_healthy = jnp.where(state.q[2] > max_z, 0.0, is_healthy)
 
-        diff_cost = -jnp.sum(jnp.square(self.sys.qpos0 - state.qpos))
+        diff = self.sys.qpos0 - state.qpos
+        original_pos_reward = jnp.exp(-2 * jnp.linalg.norm(diff)) - 0.2 * jnp.clip(jnp.linalg.norm(diff), 0, 0.5)
```
Comment (budzianowski) [inline environment.py]: create config at the top with these hyper parameters.

File: environment.py [hunk 3]
```diff
@@ -221,7 +223,7 @@ def compute_reward(
 
         # jax.debug.print("diff {}", diff_cost)
 
-        total_reward = 0.1 * ctrl_cost + 0.1 * diff_cost + 5 * is_healthy
+        total_reward = 0.1 * ctrl_cost + 1 * original_pos_reward + 5 * is_healthy
```
Comment (budzianowski) [inline environment.py]: move the hyperparameters up

File: environment.py [hunk 4]
```diff
@@ -252,6 +254,23 @@ def get_obs(self, data: MjxState, action: jnp.ndarray) -> jnp.ndarray:
             data.qfrc_actuator,
         ]
 
+        # qpos_component = data.qpos
```
Comment (budzianowski) [inline environment.py]: remove

File: convert.py [hunk 5]
```diff
@@ -1,95 +1,71 @@
+import os
```
Comment (budzianowski) [inline convert.py]: Add readme what this file is for
